proteus/NumericalSolution.py

```diff
@@ -270,13 +270,15 @@ def __init__(self,so,pList,nList,sList,opts,simFlagsList=None):
                     tmesh = TriangleTools.TriangleBaseMesh(baseFlags=n.triangleOptions,
                                                         nbase=1,
                                                         verbose=10)
+                    # ARB Before merge figure out the best way to deal with problem (eg in parallel
```

Comment:
This is fixed by the simple change below, right? Just read the mesh on all processors so it is consisent?  We could raise an issue on refactoring the 2D mesh generation with triangle, gmsh, etc. and do this all better in the refactor.

Reply:
/cc @zhang-alvin